lighthouse-core/audits/errors-in-console.js

```diff
@@ -41,11 +43,43 @@ class ErrorLogs extends Audit {
     };
   }
 
+  /** @return {AuditOptions} */
+  static defaultOptions() {
+    return {
+      ignoredPatterns: undefined,
```

Comment:
i forget what we prefer type-wise.. should this be `null`?

Reply:
it's `null` for all of our constants that end up in `config.settings` because we want to distinguish between the default and unset state, but here I don't think we have a preference. willing to accept whatever.

Reply:
any reason to do explicit `undefined` over optional?

Reply:
not really other than a more legible version of the typedef for documentation, will remove